### PR TITLE
New tutorial: data management in medicinal chemistry

### PR DESCRIPTION
**Pull request by wee-snufkin**
So happy to share my first contribution to the computational chemistry topic! 
It's an entry-level tutorial explaining the concepts of my beloved medicinal chemistry and it's combined with the FAIR data management and some basic analyses.

Thanks for the ELIXIR support @kkamieniecka & @poterlowicz-lab !

### CODE REVIEW

**Comment by simonbray** [comment]
Looks very nice. I see there are a few typos, would you mind running a spell checker over it?

**Review by simonbray — APPROVED** [review]
I'm afraid I don't have time to run through the hands-on parts myself, but with that caveat, this looks good, I'm happy for it to be merged :)

File: topics/computational-chemistry/tutorials/med-chem-data/tutorial.md [hunk 1]
```diff
@@ -357,18 +358,18 @@ Since we spoke about the drug-likeness in the previous section, let's see how it
 >
 > 1. {% tool [Drug-likeness](toolshed.g2.bx.psu.edu/repos/bgruening/qed/ctb_silicos_qed/2021.03.4+galaxy0) %} with the following parameters:
 >    - {% icon param-file %} *"Molecule data in SDF or SMILES format"*: `SDF output for Reaction maker`
->    - *"Method"* - two possible methods to weight the features are available - one based on max. weight (QEDw,max), the other on mean weight (QEDw,mo). There is also an option to leave features unweighted (QEDw,u) and we'll choose this one: `unweighted (QEDw,u)`
+>    - *"Method"* - two possible methods to weigh the features are available - one based on max. weight (QEDw,max), and the other on mean weight (QEDw,mo). There is also an option to leave features unweighted (QEDw,u) and we'll choose this one: `unweighted (QEDw,u)`
```
Comment (simonbray) [inline topics/computational-chemistry/tutorials/med-chem-data/tutorial.md:361]: I think the original `weight` was correct here

**Comment by wee-snufkin** [comment]
> Looks very nice. I see there are a few typos, would you mind running a spell checker over it?

Thanks! Apologies for the typos, I've just applied the corrections from a spell checker :)

File: topics/computational-chemistry/tutorials/med-chem-data/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,466 @@
...
+      - data-management
+   
+
+follow_up_training:
+  -
+    type: "internal"
+    topic_name: computational-chemistry
+---
+
+# Introduction
```
Comment (hexylena) [inline topics/computational-chemistry/tutorials/med-chem-data/tutorial.md:51]:
```suggestion
```

**Review by hexylena — APPROVED** [review]

**Comment by hexylena** [comment]
Great work @wee-snufkin ! If you'd like to announce this in a news item, please feel free to submit a PR for that :)

**Pull request merged**